doctests.py

```diff
@@ -104,27 +104,36 @@ def __init__(self, return_code, output):
     def result_text(self):
         return "ERROR"
 
-
-def run_test(args):
+def check_test(proc):
     try:
-        check_output(args, stderr=STDOUT, timeout=5)
-    except CalledProcessError as e:
-        return CommandError(e.returncode, e.output.decode('utf-8'))
+        out, err = proc.communicate(timeout=5)
+        if proc.returncode != 0:
+            return CommandError(proc.returncode, out)
```

Comment:
```suggestion
            return CommandError(proc.returncode, out.decode('utf-8'))
```

Means it prints a formatted stacktrace
```
INFO:TOWER:RECEIVED: Unassigned bell '7'
INFO:TOWER:RECEIVED: Unassigned bell '8'
INFO:TOWER:Disconnect
Traceback (most recent call last):
  File "./run-wheatley", line 15, in <module>
    wheatley.main.main(args, stop_on_join_tower)
  File "D:\Development\wheatley\wheatley\main.py", line 478, in main
    console_main(unparsed_args, stop_on_join_tower)
  File "D:\Development\wheatley\wheatley\main.py", line 457, in console_main
    tower.wait_loaded()
``` 
instead of on one line
```
INFO:TOWER:RECEIVED: Unassigned bell \'8\'\r\nINFO:TOWER:Disconnect\r\nTraceback (most recent call last):\r\n  File "./run-wheatley", line 15, in <module>\r\n    wheatley.main.main(args, stop_on_join_tower)\r\n  File "D:\\Development\\wheatley\\wheatley\\main.py", line 478, in main\r\n ```

Reply:
That looks like a good shout. Definitely should go in.

Reply:
Please take a peek when you have a mo'

Reply:
Prints nicely now, thanks